extensions/amp-base-carousel/0.1/test-e2e/test-arrows-non-looping.js

```diff
@@ -45,27 +42,30 @@ describes.endtoend('AMP carousel arrows when non-looping', {
   beforeEach(async() => {
     controller = env.controller;
 
-    prevArrowSlot = await getPrevArrowSlot(controller);
-    nextArrowSlot = await getNextArrowSlot(controller);
+    prevArrow = await getPrevArrow(controller);
     nextArrow = await getNextArrow(controller);
   });
 
   it('should have the arrows in the correct initial state', async() => {
-    await expect(css(prevArrowSlot, 'opacity')).to.equal('0');
-    await expect(css(nextArrowSlot, 'opacity')).to.equal('1');
+    await expect(css(prevArrow, 'opacity')).to.equal('0');
+    await expect(css(nextArrow, 'opacity')).to.equal('1');
   });
 
   it('should show the prev arrow when going to the first slide', async() => {
     await controller.click(nextArrow);
-    await expect(css(prevArrowSlot, 'opacity')).to.equal('1');
-    await expect(css(nextArrowSlot, 'opacity')).to.equal('1');
+    await expect(css(prevArrow, 'opacity')).to.equal('1');
+    await expect(css(nextArrow, 'opacity')).to.equal('1');
   });
 
   it('should hide the next arrow when going to the end', async() => {
-    const el = await getScrollingElement(controller);
-    await controller.scrollBy(el, {left: SLIDE_COUNT * pageWidth});
```

Comment:
If scrollBy isn't working, would like to fix that since a lot of tests rely on it.

Reply:
Agreed and it looks like a lot of those tests are either skipped or failing. OK to focus on that in a future PR since this test checks arrow opacity?

Reply:
Could add a TODO to put it back, and leave the existing code commented out. Another problem with the click approach is that it will make the test take longer than necessary.

Reply:
Sure thing, added it back with a TODO. Also decreased the slide count from 7 to 4 so this last test takes ~1s instead of ~1.5s